experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -170,6 +167,7 @@ This uses the `ClimaAtmos.jl` driver, with parameterization options specified in
 ## init atmos model component
 include("components/atmosphere/climaatmos_init.jl")
 atmos_sim = atmos_init(FT, parsed_args);
+thermo_params = get_thermo_params(atmos_sim) # TODO: this should be shared by all models
```

Comment:
Is there a plan for this TODO to be addressed soon(ish)?

Reply:
Not immediately, but we could think about this soon - this will depend on how we want to deal with general parameter passing, so we'll need to make sure all of the dependent teams (especially, Atmos, Land, ML, and eventually Ocean) are onboard and modify their interface accordingly. I've opened an [issue](https://github.com/CliMA/ClimaCoupler.jl/issues/342) to track this.